doc/cephfs/index.rst

```diff
@@ -103,8 +107,9 @@ Mounting CephFS
 
     Client Configuration Settings <client-config-ref>
     Client authentication <client-auth>
-    Mount CephFS using Kernel Driver <kernel>
-    Mount CephFS using FUSE <fuse>
+    Mount CephFS <mount>
```

Comment:
Let's rename this `Mount Preparation` or something.

Reply:
Renamed it to "Mount CephFS: Prerequisites". I named it so because I thought a title similar to "Mount CephFS using Kernel" and "Mount CephFS using FUSE" would sound better.